x-pack/plugins/remote_clusters/__jest__/client_integration/remote_clusters_add.test.js

```diff
@@ -77,7 +77,7 @@ describe('Create Remote cluster', () => {
       beforeEach(async () => {
         ({ component, form, actions } = setup());
 
-        await nextTick();
+        await nextTick(100);
```

Comment:
I needed to adjust the timeout when switching from `axios` to the http service

Reply:
Was the issue the same in each instance where nextTick was added? Just wondering whether it is worth putting a comment in each time.

Not a big deal though :)

Reply:
Good point! I may have gotten carried away 😂. It actually is not needed for this specific line, but I updated the code where it was needed with a comment.